tests/unit/test_helper.py

```diff
@@ -36,12 +37,61 @@ def test_handle_enums():
 
 
 def test_flowpod_to_namespace():
-    pass
+    pod_dict = {
+        'head': {
+            'uses': '_route',
+            'parallel': 1
+        },
+        'tail': {
+            'uses': '_reduce',
+            'parallel': 1
+        },
+        'peas': [{
+            'uses': 'Encoder',
+            'pea_id': 1
+        },
+            {
+                'uses': 'Indexer',
+                'pea_id': 2
+            },
+        ]
+    }
+
+    pod_args = flowpod_to_namespace(pod_dict)
+
+    assert len(pod_args) == 3
+    head_args = pod_args['head']
+    assert 'log_id' in head_args
+    assert head_args.parallel == 1
+    assert head_args.uses == '_route'
+
+    tail_args = pod_args['tail']
+    assert 'log_id' in tail_args
+    assert tail_args.parallel == 1
+    assert tail_args.uses == '_reduce'
+
+    assert len(pod_args['peas']) == 2
+    pea_1 = pod_args['peas'][0]
+    assert 'log_id' in pea_1
+    assert pea_1.uses == 'Encoder'
+    assert pea_1.pea_id == 1
+
+    pea_2 = pod_args['peas'][1]
+    assert 'log_id' in pea_2
+    assert pea_2.uses == 'Indexer'
+    assert pea_2.pea_id == 2
 
 
 def test_basepod_to_namespace():
-    pass
+    pod_args = basepod_to_namespace(PodModel())
+    assert 'identity' in pod_args
+    assert 'port_expose' in pod_args
+    assert 'uses' in pod_args
 
 
 def test_basepea_to_namespace():
-    pass
+    pea_args = basepea_to_namespace(PeaModel())
+    assert 'identity' in pea_args
+    assert 'port_expose' in pea_args
+    assert 'uses' in pea_args
+    assert 'py_modules' in pea_args
```

Comment:
One `assert not` for an arg that is available in pod_args, but not in pea_args (since it is a subset)?

Reply:
any idea of a candidate?

Reply:
`parallel` - https://github.com/jina-ai/jina/blob/master/jina/parser.py#L411